index.ts

```diff
@@ -16,8 +16,11 @@ const program = new Command(packageJson.name)
   .option('-d, --dry', 'Dry run (no changes are made to files)')
   .option('-p, --print', 'Print transformed files to stdout')
   .option('--verbose', 'Show more information about the transform process')
+  .option('--silent', "Don't print anything to stdout")
   .usage('[codemod] [source] [options]')
-  .action(transform)
+  .action((codemodName, source, options) => {
+    transform(codemodName, source, options)
+  })
```

Comment:
This change seems to be redundant

Reply:
No, because now the transform function returns a value, and .action should receive a function that returns void.

Reply:
I have applied changes to this

Reply:
Shouldn't we check if there were any files that changed? That's why I wanted it to return what the runner was giving back.

Reply:
We are checking what's being passed to the `run` method from `jscodeshift`. We expect to give a certain format. We have reached a border between our code and someone else code. It's fine for the unit test to end here.

It would be great to set up some integration tests to test our code from start to end at some later stage. We can discuss it as we can do it in jest or for example playwright. Before that, there are a couple of topics to solve tho. I have 2 on my mind for now. One, those tests would require more time to execute and might require some different setup for jest (if we decide to use jest). Two, we need some dedicated test directory to run tests on them.

Reply:
Sure, I think that's the next thing we're going to do